lib/config/definitions.ts

```diff
@@ -525,6 +526,18 @@ const options: RenovateOptions[] = [
     mergeable: true,
     cli: false,
   },
+  {
+    name: 'aliases',
+    description: 'Aliases for registries, package manager specific',
+    type: 'object',
+    default: {
+      stable: 'https://kubernetes-charts.storage.googleapis.com/',
```

Comment:
I think let's move this default inside the Helm default config, not the global `aliases` default

Reply:
How to do that?

Reply:
Add an `aliases` array inside the `helm-requirements` default config object in this file, and remove this `default`.

Reply:
I've moved this now but you'll need to clean up some things locally (e.g. the json schema regeneration)